the-real-instgram/src/components/PostsPage.js

```diff
@@ -18,14 +18,24 @@ class PostPage extends React.Component{
           stateArray: dummyData
         })
       }
-      filterPosts = post =>{
-        post.preventDefault();
-        console.log("filter!!!")
+
+      handleSearchInput=event=>{    
+        event.preventDefault()
+        console.log(event.target.value)
```

Comment:
great job console logging!
you can get this to work. let me know if you need help!